tests/test-maximize.lua

```diff
@@ -82,10 +82,11 @@ local steps = {
         local bw      = c.border_width
 
         assert(c.fullscreen)
-        assert(new_geo.x-bw==sgeo.x)
-        assert(new_geo.y-bw==sgeo.y)
-        assert(new_geo.x+new_geo.width+bw==sgeo.x+sgeo.width)
-        assert(new_geo.y+new_geo.height+bw==sgeo.y+sgeo.height)
+
+        assert(new_geo.x==sgeo.x)
+        assert(new_geo.y==sgeo.y)
+        assert(new_geo.x+new_geo.width+2*bw==sgeo.x+sgeo.width)
+        assert(new_geo.y+new_geo.height+2*bw==sgeo.y+sgeo.height)
```

Comment:
That code path wasn't of my making ;) (but the test is). Actually, after thinking about it, it was probably done on purpose. Done this way, you get "real" fullscreen. This was initially why I left it that way.  But now it conflicts with the request API, so it has to use the standard code path or these issues will keep piling up.